Libplanet.Tests/Blockchain/BlockChainTest.cs

```diff
@@ -2,6 +2,7 @@
 using System.Collections.Generic;
 using System.Collections.Immutable;
 using System.Linq;
+using System.Reflection;
```

Comment:
Is this actually used?

Reply:
it seems to be needed to using `TargetInvocationException`.

Reply:
Yes, it is used for `TargetInvocationException`.